src/getting_grain.md

```diff
@@ -112,21 +112,26 @@ This section contains solutions to problems which people may run into when build
 
 #### After install, the `grain` command is not found
 
-This likely means that you haven't put `yarn`'s `bin` directory on your shell's `PATH` variable. This can be fixed by setting the variable in your shell's configuration. For example, if you use `zsh`, you can do the following:
+This likely means that you haven't put `npm`'s global `bin` directory on your shell's `PATH` variable. This can be fixed by setting the variable in your shell's configuration. For example, if you use `zsh`, you can do the following:
+
 ```bash
 # Initialize the PATH variable in your configuration and enable the change
-(echo 'export PATH="${PATH}:$(yarn global bin)"' >> ~/.zshrc) && source ~/.zshrc
+(echo 'export PATH="${PATH}:$(npm bin -g)"' >> ~/.zshrc) && source ~/.zshrc
```

Comment:
I like to have the flags fully written out for examples. I feel like it's helpful for some people.
```suggestion
(echo 'export PATH="${PATH}:$(npm bin --global)"' >> ~/.zshrc) && source ~/.zshrc
```